doc/latex.rst

```diff
@@ -1565,30 +1568,34 @@ Macros
      ``\sphinxstyleliteralintitle``;   ``\sphinxcode{#1}``
      ``\sphinxstylecodecontinued``;    ``{\footnotesize(#1)}}``
      ``\sphinxstylecodecontinues``;    ``{\footnotesize(#1)}}``
-     ``\sphinxstylenotetitle``; ``\sphinxdotitlerowwithicon{note}{#1}``
-     ``\sphinxstylehinttitle``; ``\sphinxdotitlerowwithicon{hint}{#1}``
-     ``\sphinxstyleimportanttitle``;   *similar*
-     ``\sphinxstyletiptitle``;         *similar*
-     ``\sphinxstylewarningtitle``;     *similar*
-     ``\sphinxstylecautiontitle``;     *similar*
-     ``\sphinxstyleattentiontitle``;   *similar*
-     ``\sphinxstyledangertitle``;      *similar*
-     ``\sphinxstyleerrortitle``;       *similar*
-     ``\sphinxstyleseealsotitle``;     *similar*
-     ``\sphinxstyletodotitle``;        *similar*
+     ``\sphinxstylenotetitle``;        ``\sphinxdotitlerow{note}{#1}``
+     ``\sphinxstylehinttitle``;        ``\sphinxdotitlerow{hint}{#1}``
+     ``\sphinxstyleimportanttitle``;   ``\sphinxdotitlerow{important}{#1}``
+     ``\sphinxstyletiptitle``;         ``\sphinxdotitlerow{tip}{#1}``
+     ``\sphinxstylewarningtitle``;     ``\sphinxdotitlerow{warning}{#1}``
+     ``\sphinxstylecautiontitle``;     ``\sphinxdotitlerow{caution}{#1}``
+     ``\sphinxstyleattentiontitle``;   ``\sphinxdotitlerow{attention}{#1}``
+     ``\sphinxstyledangertitle``;      ``\sphinxdotitlerow{danger}{#1}``
+     ``\sphinxstyleerrortitle``;       ``\sphinxdotitlerow{error}{#1}``
+     ``\sphinxstyleseealsotitle``;     ``\sphinxdotitlerow{seealso}{#1}``
+     ``\sphinxstyletodotitle``;        ``\sphinxdotitlerow{todo}{#1}``
+     ``\sphinxstyletopictitle``;       ``\sphinxdotitlerow{topic}{#1}``
+     ``\sphinxstylecontentstitle``;    ``\sphinxdotitlerow{contents}{#1}``
+     ``\sphinxstylesidebartitle``;     ``\sphinxdotitlerow{sidebar}{#1}``
 
   .. note::
 
      To let this table fit on the page width in PDF output we have lied a bit
-     and the actual definition of ``\sphinxstylenotetitle`` is:
+     and the actual definition of, for example, ``\sphinxstylenotetitle`` is:
```

Comment:
```suggestion
     For instance, the actual definition of ``\sphinxstylenotetitle`` is:
```

\+ add a period after the `lied a bit` just before.

Reply:
note to myself "don't forget to add the period" after commiting suggestions